ecl.yaml

```diff
@@ -0,0 +1,68 @@
+name: ecl
+config-opts:
+  - --enable-unicode=yes
+  - --enable-c99complex
+  - --enable-threads=yes
+  - --with-__thread
+  - --with-clx
+  - --disable-rpath
+build-options:
+  arch:
+    i386:
+      config-opts:
+        - --with-sse=no
+    x86_64:
+      config-opts:
+        - --with-sse=auto
+sources:
+  - type: archive
+    url: https://common-lisp.net/project/ecl/static/files/release/ecl-16.1.3.tgz
+    sha256: 76a585c616e8fa83a6b7209325a309da5bc0ca68e0658f396f49955638111254
+  # This patch was sent upstream on 4 Feb 2012. 
+  # It fixes a few warnings from the C compiler
+  # that indicate situations that might be dangerous at runtime
+  # https://src.fedoraproject.org/rpms/ecl/raw/79105db8cb5c6b79766dd06de9e816186b2e2099/f/ecl-16.1.3-warnings.patch
+  - type: patch
+    path: ecl-16.1.3-warnings.patch
+    strip-components: 0
+  # Do not use a separate thread to handle signals by default
+  # if built with boehm-gc support.
+  # This prevents a deadlock when building maxima with ecl support,
+  # and should handle by default these problems:
+  # http://trac.sagemath.org/sage_trac/ticket/11752
+  # http://www.mail-archive.com/ecls-list@lists.sourceforge.net/msg00644.html
+  # https://src.fedoraproject.org/rpms/ecl/raw/79105db8cb5c6b79766dd06de9e816186b2e2099/f/ecl-16.1.3-signal_handling_thread.patch
+  - type: patch
+    path: ecl-16.1.3-signal_handling_thread.patch
+    strip-components: 0
+  # GCC does not implement support for #pragma STDC FENV_ACCESS
+  # https://src.fedoraproject.org/rpms/ecl/raw/79105db8cb5c6b79766dd06de9e816186b2e2099/f/ecl-16.1.3-fenv-access.patch
+  - type: patch
+    path: ecl-16.1.3-fenv-access.patch
+    strip-components: 0
+  # fix when building with -Werror=format-security
+  # https://src.fedoraproject.org/rpms/ecl/raw/79105db8cb5c6b79766dd06de9e816186b2e2099/f/ecl-16.1.3-end_of_line.patch
+  - type: patch
+    path: ecl-16.1.3-end_of_line.patch
+    strip-components: 0
+  # Upstream patch to fix the SSE printer
+  # https://src.fedoraproject.org/rpms/ecl/raw/79105db8cb5c6b79766dd06de9e816186b2e2099/f/ecl-16.1.3-sse-printer.patch
+  - type: patch
+    path: ecl-16.1.3-sse-printer.patch
+    strip-components: 0
+  # Upstream patch to fix maxima test failure with atan with signed zero
```

Comment:
Are tests executed at all during build or normal run?

Reply:
> Are tests executed at all during build or normal run?

If I remember correctly, this is actually a patch for the atan2 function (2-argument arctangent).